lake/ztests/index/apply-by-ip.yaml

```
@@ -6,20 +6,32 @@ script: |
   zed lake index create -q IPs type ip
   zed lake index apply -use test@main IPs $id
   echo ===
-  zed lake query -Z "from test@main:rawlog | cut this:=object.Rule | drop id,ts"
+  zed lake query -Z -I query.zed
 
 inputs:
   - name: in.zson
     data: |
       {x:127.0.0.1}
       {x:127.0.0.2}
+  - name: query.zed
+    data: |
+      from (
+        :index_rules => sort id;
+        test@main:indices => sort rule.id | cut o:=this;
+      )
+      | left join on id = o.rule.id
+      | count() by name,type
+      | sort name
```

This works without the cut or the sort.
```suggestion
      from (
        :index_rules => sort id;
        test@main:indices => sort rule.id;
      )
      | left join on id = rule.id
      | count() by name,type
```

---

I had this but it was passing intermittently for me. It wasn't doing this for you?

---

I have `while make TEST=TestZed/lake/ztests/index/apply-by-ip; do :; done` running with it and haven't seen any failures.

---

Oh snap, duh this only returns once record.